File: app/templates/server_go/gin/api/todo/route/todoroute.go
package todoroute

import (
	"<%= repoHostUrl %>/<%= userNameSpace %>/<%= appName %>/server/api/todo/controller"
	"github.com/gin-gonic/gin"
)

func Init(r *echo.Echo) {
	r.GET("/api/todos", todocontroller.GetAll)
	r.POST("/api/todos", todocontroller.NewTodo)
	r.DELETE("/api/todos/:id", todocontroller.RemoveTodo)
}

File: app/templates/server/node/express/server_node_express/api/todo/route/todo-route.js
"use strict";

const TodoController = require("../controller/todo-controller");

module.exports = class TodoRoutes {
    static init(router) {
      router
        .route("/api/todos")
        .get(TodoController.getAll)
        .post(TodoController.createTodo);

      router
        .route("/api/todos/:id")
        .delete(TodoController.deleteTodo);
    }
}
